### PR TITLE
Feat/My Seedlots bug fixes (1/2)

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description

These changes in My Seedlot screen will be implemented in this issue:

Subtitle wrong should be "Check and manage your own seedlots"
Column wrong should be "Seedlot class", "Seedlot species" and "Last updated"
Hover doesn't change to clickable object on My seedlots table
Icons go/back next page on My seedlots page/table are misaligned
Background color on "table title" is wrong on My seedlots page
Line color above "table title" is wrong
Use constants.ts

Fixes # [FSADT2-616](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-616)
 [Github issue](https://github.com/bcgov/nr-spar/issues/179)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-180-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-180-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-180-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/src/views/Seedlot/MySeedlots/SeedlotDataTable/headerData.ts [hunk 1]
```diff
@@ -5,11 +5,11 @@ const headerData = [
   },
   {
     key: 'class',
-    header: 'Lot class'
+    header: 'Seedlot class'
```
Comment (mgaseta) [inline frontend/src/views/Seedlot/MySeedlots/SeedlotDataTable/headerData.ts:8]: Nice work here, but I think this headerData var should be in the `constants.ts` file, not on it's own.

File: frontend/src/views/Seedlot/MySeedlots/styles.scss [hunk 2]
```diff
@@ -21,3 +21,18 @@
   margin: 0;
```
Comment (mgaseta) [inline frontend/src/views/Seedlot/MySeedlots/styles.scss:21]: Use the bcgov prefix var on this file.

File: frontend/src/views/Seedlot/MySeedlots/styles.scss [hunk 3]
```diff
@@ -21,3 +21,18 @@
...
+
+.bcgov--search-input { 
+  border-bottom: 1px solid var(--bcgov-border-subtle-01)
+}
+
+.bcgov--data-table tr{
+  cursor: pointer;
+}
+
+.bcgov--pagination__button, .bcgov--btn--ghost.bcgov--pagination__button{
```
Comment (mgaseta) [inline frontend/src/views/Seedlot/MySeedlots/styles.scss:36]: Break the line between identifiers.

File: frontend/src/views/Seedlot/MySeedlots/constants.ts [hunk 4]
```diff
@@ -1,7 +1,45 @@
...
+    },
+    {
+      key: 'last_modified',
+      header: 'Last updated'
+    },
+    {
+      key: 'approved_at',
+      header: 'Approved at'
+    }
+  ]
```
Comment (mgaseta) [inline frontend/src/views/Seedlot/MySeedlots/constants.ts:42]: This can be a separate variable, don't need to be inside `tableText`.

**Review by RMCampos — APPROVED** [review]
Good job! 💯

**Review by mgaseta — APPROVED** [review]
LGTM! 🚀🎉

**Pull request merged**